util/test.js

```diff
@@ -7,13 +10,37 @@ const getCode = async (bundle, outputOptions, allFiles = false) => {
 
   if (allFiles) {
     return output.map(({ code, fileName, source, map }) => {
-      return { code, fileName, source, map };
+      return {
+        code,
+        fileName,
+        source,
+        map
+      };
     });
   }
   const [{ code }] = output;
   return code;
 };
 
+/**
+ * @param {import('rollup').RollupBuild} bundle
+ * @param {import('rollup').OutputOptions} [outputOptions]
+ */
+const getFiles = async (bundle, outputOptions) => {
+  if (!outputOptions.dir && !outputOptions.file)
+    throw new Error('You must specify "output.file" or "output.dir" for the build.');
```

Comment:
If `output.file` isn't specified, can we figure it out based on input? Since Rollup doesn't require this option I want to make sure our tests support that case too

Reply:
Well rollup requires either `output.file` or `output.dir` so throwing is valid, moreover this is just copied from rollup.
We do not write files so this emulates the rollup code for writing assets, see also this line:
https://github.com/rollup/rollup/blob/b179695aab154bcb9b7e35e0a16f9afd7f1264f1/src/rollup/rollup.ts#L166